cmd/meroxa/turbine_cli/javascript/build.go

```diff
@@ -13,7 +12,8 @@ import (
 func Build(ctx context.Context, l log.Logger, path string) error {
 	// TODO: Handle the requirement of https://github.com/meroxa/turbine-js.git being installed
 	// cd into the path first
-	cmd := exec.Command("npx", "--yes", "@meroxa/turbine-js@0.1.7", "test", path)
+	isDevModeActive := GetDevModeSetting()
```

Comment:
Here I'm currently running into the error `GetDevModeSetting` is undefined -- do I need to import this util from the `global` package first?

Reply:
It seems `global.GetDevModeSetting` indeed fixed it